client/Reviews.jsx

```diff
@@ -27,7 +27,7 @@ const Reviews = ({ reviews }) => {
           {review.city}
         </div>
         <div className="user-num-reviews">
-          <img className="msg-icon" src={msgIcon} alt="icon" />
+          <img className="msg-icon" src="images/781653508da1149f188dadf547aa46a7.png" alt="icon" />
```

Comment:
Super interesting way of bringing the image source in. If I had to be picky I'd say that almost belongs in a .env.